oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/flatfile/FlatFileNodeStoreBuilder.java

```diff
@@ -222,26 +235,53 @@ public List<FlatFileStore> buildList(IndexHelper indexHelper, IndexerSupport ind
         return storeList;
     }
 
-    private File createdSortedStoreFile() throws IOException, CompositeException {
+    /**
+     * Returns the existing list of store files if it can read from system property OAK_INDEXER_SORTED_FILE_PATH which 
+     * defines the existing folder where the flat file store files are present. Will throw an exception if it cannot 
+     * read or the path in the system property is not a directory.
+     * If the system property OAK_INDEXER_SORTED_FILE_PATH in undefined, or it cannot read relevant files it 
+     * initializes the flat file store.
+     * 
+     * @return list of flat files
+     * @throws IOException 
+     * @throws CompositeException
+     */
+    private List<File> createdSortedStoreFiles() throws IOException, CompositeException {
+        // Check system property defined path
         String sortedFilePath = System.getProperty(OAK_INDEXER_SORTED_FILE_PATH);
```

Comment:
(I have to admit I don't understand why we use a system property, and not a parameter... but this is not the fault of this patch, so let's keep that for now.)

Reply:
FWFW, if it needs to be a system property, it should use https://jackrabbit.apache.org/oak/docs/apidocs/org/apache/jackrabbit/oak/commons/properties/SystemPropertySupplier.html

Reply:
SystemPropertySupplier is nice because it logs if the value is set (to a non-default value). However in this case I think not using it is fine.